variables.tf

```diff
@@ -397,6 +397,11 @@ variable "create_nfs_public_ip" {
   default = null
 }
 
+variable "use_single_subnet_for_workers" {
+  type    = bool
+  default = false 
+}
+
```

Comment:
Not sure this is needed plus the name is a bit off.  Work with @manoatsas to make sure the logic is cleaned up here.

Reply:
Add description with the trade off warning, that cluster nodes will be restricted to single AZ and not HA.